src/main/webapp/script.js

```diff
@@ -0,0 +1,37 @@
+function countUnits(){
+  const inputElems = document.getElementsByName("search-units");
+  const arr = []
+  for (let i=0; i<inputElems.length; i++) {       
+    if (inputElems[i].checked){
+      arr.push(inputElems[i].value);
+    }
+  }
+  return arr;
+}
+
+function showClasses() {
+  const classes = document.getElementById("search-results");
```

Comment:
This variable should be more specific considering all the class terminology in this method. Maybe something like "classResults" or "classResultsContainer"